packages/test-base/src/androidTest/kotlin/io/realm/kotlin/test/shared/dynamic/DynamicMutableRealmTests.kt

```diff
@@ -642,4 +642,25 @@ class DynamicMutableRealmTests {
             }
         }
     }
+
+    @Test
+    fun deleteAll() {
```

Comment:
For dynamic Realms, I would also test the case where the schema changes, i.e. what happens if you have a realm with data for `schema = setOf(Person, Dog)` but then open the same Realm with `schema = setOf(Person)`..what happens with the dog elements? (I know the answer, but it would be nice with a test for it).